modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Project/Routines/Builds/BuildForm/BuildFormCases.tsx

```diff
@@ -92,28 +70,35 @@ const BuildFormCases: React.FC<BuildFormCasesProps> = ({
 
 	return (
 		<>
-			<h3>{i18n.translate('cases')}</h3>
+			<h3>{i18n.translate(title)}</h3>
```

Comment:
pass the translated prop

```suggestion
			<h3>{title}</h3>
```